salt/modules/k8s.py

```diff
@@ -130,6 +200,27 @@ def get_labels(node=None, apiserver_url=None):
     return {"labels": ret}
 
 
+def get_namespaces(namespace="", apiserver_url=None):
+    '''
+    Get k8s namespaces
+
```

Comment:
Can you please add a `.. versionadded:: Carbon` directive here? As well as some docs explaining what the kwargs are?